base_delivery_carrier_files/__manifest__.py

```diff
@@ -21,9 +21,9 @@
 
 {
     'name': 'Base Delivery Carrier Files',
-    'version': '1.2.3',
+    'version': '10.0.1.0.1',
     'category': 'Generic Modules/Warehouse',
-    'description': """
+    'summary': """
```

Comment:
This is way too long for a summary :-)

![image](https://user-images.githubusercontent.com/1466356/56673183-1989fd00-66b8-11e9-8b48-b43c4a9a4220.png)